ProcessLib/RichardsFlow/RichardsFlowFEM.h

```diff
@@ -96,6 +108,11 @@ class LocalAssemblerData : public RichardsFlowLocalAssemblerInterface
             _integration_method.getNumberOfPoints();
         SpatialPosition pos;
         pos.setElementID(_element.getID());
+
```

Comment:
@wenqing I changed the code according to your suggestions. Would be nice if you could have a look again.